### PR TITLE
Review of template imports

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

**Review by olieidel — APPROVED** [review]

File: src/documents/drafts/62304-mapping.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ OpenRegulatory templates).
 
 ## Document Approval
 
-| Activity | Date       | Role | Name        | Initials |
-| -------- | ---------- | ---- | ----------- | -------- |
-| Creation | 2022-02-28 | CEO  | Simon Biggs | SB       |
+| Activity | Date       | Role                  | Name             | Initials |
```
Comment (olieidel) [inline src/documents/drafts/62304-mapping.md:109]: This is a record and doesn't need to be approved, so it only needs creation which you can take from the git history. So no signing table needed.

File: src/documents/drafts/quality-manual-policy-objectives.md [hunk 2]
```diff
@@ -101,6 +101,8 @@ The Quality Management Officer (QMO) is responsible to:
 
 ## Document Approval
 
-| Activity | Date       | Role | Name        | Initials |
-| -------- | ---------- | ---- | ----------- | -------- |
-| Creation | 2022-02-28 | CEO  | Simon Biggs | SB       |
+| Activity | Date       | Role                  | Name             | Initials |
+| -------- | ---------- | --------------------- | ---------------- | -------- |
+| Creation | 2022-02-28 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel |        |
```
Comment (olieidel) [inline src/documents/drafts/quality-manual-policy-objectives.md:107]:
```suggestion
| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel | OE     |
```

File: src/documents/drafts/sop-integrated-software-development.md [hunk 3]
```diff
@@ -413,6 +413,8 @@ assigned.
 
 ## Document Approval
 
-| Activity | Date       | Role | Name        | Initials |
-| -------- | ---------- | ---- | ----------- | -------- |
-| Creation | 2022-02-28 | CEO  | Simon Biggs | SB       |
+| Activity | Date       | Role                  | Name             | Initials |
+| -------- | ---------- | --------------------- | ---------------- | -------- |
+| Creation | 2022-02-28 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel |        |
```
Comment (olieidel) [inline src/documents/drafts/sop-integrated-software-development.md:419]:
```suggestion
| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel | OE     |
```

**Review by olieidel — APPROVED** [review]

File: src/documents/drafts/62304-mapping.md (hunk 1 above)
Comment (SimonBiggs) [reply src/documents/drafts/62304-mapping.md:109]: Awesome, thanks @olieidel :slightly_smiling_face:

**Pull request merged**